main.cpp

```diff
@@ -52,84 +52,86 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       tree
+//  action 1:   the tree grows taller
+tree.growUp();
+//  action 2:   the tree grows leaves
+tree.growLeaves();
+//  action 3:   the tree sheds leaves
+tree.shedLeaves();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       leg                         
+//  action 1:   the leg extends        
+leg.extend();    
+//  action 2:   the leg flexes  
+leg.flex(); 
+//  action 3:   the leg rotates conter-clockwise
+leg.rotateCounterClockwise();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       finger
+//  action 1:   the finger extends
+finger.extend();
+//  action 2:   the finger points 
+finger.point();
+//  action 3:   the finger rotates counter-clockwise
+finger.rotateCounterClockwise();
+//  4)          
+//  Noun:       thumb
+//  action 1:   the thumb extends
+thumb.extend();
+//  action 2:   the thumb raises 
+thumb.raise();
+//  action 3:   the thumb rotates counter-clockwise
+thumb.rotateCounterClockwise();
 
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       eye
+//  action 1:   the eye sees 
+eye.see();
+//  action 2:   the eye blinks
+eye.blink();
+//  action 3:   the eye closes
+eye.close();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       girl 
+//  action 1:   the girl sings
+girl.sing();
+//  action 2:   the girl dances 
+girl.dance();
+//  action 3:   the girl smiles
+girl.smile();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.